packages/usa-icon-list/src/usa-icon-list.twig

```diff
@@ -5,16 +5,18 @@
 ] %}
 
 <ul class="{{ list_classes | join(' ') | trim }}">
-  {% for item in items %}
-  <li class="usa-icon-list__item">
-    <div class="usa-icon-list__icon {% if item.icon.color %} text-{{ item.icon.color }}{% endif %}">
-      <svg class="usa-icon{% if item.icon.utilities %} {{ item.icon.utilities }}{% endif %}" aria-hidden="true" role="img">
+  {% for key, item in items %}
+    <li class="usa-icon-list__item">
+      <div class="usa-icon-list__icon {% if item.icon.color %} text-{{ item.icon.color }}{% endif %}">
+        <svg class="usa-icon{% if item.icon.utilities %} {{ item.icon.utilities }}{% endif %}"{% if item.icon.title %} aria-labelledby="usa-icon__title_{{ key }}"{% endif %} aria-hidden="{% if item.icon.title %}false{% else %}true{% endif %}" role="img">
+        {% if item.icon.title %}<title id="usa-icon__title_{{ key }}">{{ item.icon.title }}</title>{% endif %}
```

Comment:
**Question:** Reading the provided recommendations, was there a reason we didn't take the `aria-label` approach in the "better" category?

In their example, they're using `aria-labelledby` to point to a visible paragraph element and didn't create a `title` element. I'm curious if `aria-labelledby` is the approach we would want to take if there is on screen content that would describe the icon.

My hunch is we can use `aria-label` to describe the icon without the need to create a `title` element on the page, resulting in less markup.

Reply:
Ahh ok thanks for clarifying! I'll mark this as resolved 👍